Documentation/operations/scalability/identity-relevant-labels.rst

```diff
@@ -89,10 +89,23 @@ will be used to evaluate Cilium identities:
 The above configuration would only include the following labels when evaluating
 Cilium identities:
 
-- io.kubernetes.pod.namespace=*
-- k8s-app=*
-- app=*
-- name=*
+- io.kubernetes.pod.namespace*=.*
+- k8s-app*=*
+- app*=*
+- name*=*
+
+As well as, labels with the same prefix as defined in the configuration will
+also be considered, this lists some examples of labels that would also be
+evaluated for Cilium identities:
```

Comment:
This sentence reads a bit strange. I think we can drop the "As well as" and make two sentences out of it:
```suggestion
Labels with the same prefix as defined in the configuration will
also be considered. This lists some examples of labels that would also be
evaluated for Cilium identities:
```